test/models/movie.test.js

```diff
@@ -13,10 +16,29 @@ describe('movie model', () => {
         'id',
         'title',
         'release_year',
+        'locations',
         'object'
       ]);
     });
 
+    it('serialzes with location', async () => {
+      const movie_payload = { name: 'Zodiac' };
+      const location_payload = { name: 'Bay Bridge' };
+
+      await Knex.raw('TRUNCATE movies CASCADE; TRUNCATE locations CASCADE; TRUNCATE locations_movies CASCADE;');
```

Comment:
Could you place this statement in a `beforeEach()`? We usually only delete or truncate from tables before each test case to maintain consistency rather than having such an operation within the scope of only one case.